src/QMCHamiltonians/ECPotentialBuilder.cpp

```diff
@@ -50,10 +50,12 @@ bool ECPotentialBuilder::put(xmlNodePtr cur)
     nonLocalPot.resize(ng,0);
   }
   std::string ecpFormat("table");
+  std::string NLPP_algo("default");
   std::string pbc("yes");
   std::string forces("no");
   OhmmsAttributeSet pAttrib;
   pAttrib.add(ecpFormat,"format");
+  pAttrib.add(NLPP_algo,"algo");
```

Comment:
This should be expanded to 'algorithm'.  I don't see an advantage in using the abbreviated name in the input file.